config/passport.js

```diff
@@ -0,0 +1,62 @@
+const passport = require( 'passport' )
+const User = require( '../models/users' )
+
```

Comment:
Please group library imports together at the top of the module, with no whitespace.  Project modules should be separated from library imports by whitespace:
```js
const passport = require( 'passport' )
const LocalStrategy = require( 'passport-local' )
const { Strategy, ExtractJwt } = require( 'passport-jwt' )

const User = require( '../models/users' )
```